packages/truffle-artifactor/.gitignore

```diff
@@ -4,3 +4,4 @@ config/development
 config/test
 yarn.lock
 package-lock.json
+dist
```

Comment:
I don't think these nested .gitignores do anything.

Reply:
I can remove them, but they do seem to make git ignore them when working locally.

Reply:
Oh, well, empirical observation is better than my postulatin'!